catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx

```diff
@@ -83,13 +117,17 @@ const ExpandedImage: FunctionComponent<Props> = ({
   const [canvasDeeplink, setCanvasDeeplink] = useState<
     CanvasLink | undefined
   >();
+  const [currentImageId, setCurrentImageId] = useState<string | undefined>();
 
   const workId = image?.source.id;
-  const displayTitle = detailedWork?.title ?? '';
-  const displayContributor = detailedWork?.contributors?.[0]?.agent?.label;
 
   useEffect(() => {
-    if (workId) {
+    if (workId && workId !== currentImageId) {
```

Comment:
Should `currentImageId` be in the `useEffect` dependencies?

Reply:
`workId` should pick up all needed triggers and as the goal of this is to update `currentImageId`'s value, I don't think it's needed, but happy to discuss?